src/Stratis.Bitcoin.Features.SmartContracts/ReflectionExecutor/Controllers/SmartContractsController.cs

```diff
@@ -96,7 +96,7 @@ public class SmartContractsController : Controller
         /// <param name="address">The address of the smart contract to retrieve as bytecode and C# source.</param>
         ///
         /// <returns>A response object containing the bytecode and the decompiled C# code.</returns>
-        [Route("code")]
+        [Route("api/[controller]/code")]
```

Comment:
Moving the prefix here is unfortunately necessary to get the RPC bindings to work.